x/stakeibc/keeper/community_pool.go

```diff
@@ -1,12 +1,104 @@
 package keeper
 
 import (
+	"errors"
+	"fmt"
+
+	sdkmath "cosmossdk.io/math"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v14/utils"
 )
 
+// For each hostZone with a valid community pool, trigger the ICQs and ICAs to transfer deposited tokens to holding address
+// Since ICQs and ICAs take time to complete, it is almost certain tokens swept in and processed will be swept out in a later epoch
+func (k Keeper) SweepInAllDepositedCommunityPoolTokens(ctx sdk.Context) error {
```

Comment:
Our hooks generally do not return an error cause they're run in the begin/end block, so you can remove the error return type here. And then make sure to still catch the errors from `ICQCommunityPoolDepositICABalances` and `IBCReturnAllCommunityPoolTokens`, but you can just log them